explainer.md

```diff
@@ -169,11 +165,11 @@ The following code snippet uses the Cookie Store API instead, and does not jank
 the main thread.
 
 ```javascript
-document.getElementById('opt-out-button').addEventListener('click', () => {
+document.getElementById('opt-out-button').addEventListener('click', async () => {
   await cookieStore.set({ name: 'opt_out', value: '1',
                           expires: new Date('Wed, 1 Jan 2025 00:00:00 GMT') });
```

Comment:
FYI, this is implicitly doing a Date &rarr; Number conversion... which should do the right thing. But we may want to consider making this a number (e.g. `Date.now() + 1000*60*60*24*365`)  instead.

Also, 2025 will be in the past at some point.

Reply:
Yup :)

This example is intended to match the `document.cookie` setter example above. I was fully aware of the implicit conversion, and wanted to (perhaps misleadingly?) point out that it should be easy to convert code that uses the `document.cookie` setter to the new API.

Do you think the concern above overrides the benefit of matching the `document.cookie` example? If so, do you think the `document.cookie` example should be updated to use a computed date as well, to match?